test/test_manifest.rb

```diff
@@ -507,8 +507,8 @@ def teardown
     manifest.find("coffee.js").each do |asset|
       paths << asset.logical_path
     end
-    assert paths.include?("coffee.js")
-    assert !paths.include?("coffee/index.js")
+    assert paths.include?("coffee/index.js")
+    assert !paths.include?("coffee.js")
```

Comment:
Hmm, if people are doing something complicated like `config.assets.precompile = [/coffee.js/]` where `coffee/index.js`, they may have to change that to `config.assets.precompile = [/coffee\/index.js/]`